### DISCUSSION TITLE
Asus Control Interface

### DISCUSSION THREAD

**Discussion by seerge** [seerge/g-helper#766] [General]
Hello fellow community,

I just noticed that Asus seem to have pulled back their latest Asus Control Interface (a "driver" that works as a proxy between bios / Armoury (or G-helper). Before it was 3.1.14 , now latest version that you can get on website / or via MyASUS / G-Helper is 3.1.13 again

![image](https://github.com/seerge/g-helper/assets/5920850/e2e97edd-289b-4cc5-82c8-66dbb038a34d)

Did you notice any changes in 14 vs 13 ?

**Comment by Elite49** [comment]
I'm still seeing the 14 version online and through G-Helper. Haven't noticed any differences, on a ROG Strix G16 (2023) G614

![image](https://github.com/seerge/g-helper/assets/138760631/210a44ea-f1ad-4790-b0fe-4e3360483652)

**Comment by seerge** [comment]
[GHelper.zip](https://github.com/seerge/g-helper/files/12088758/GHelper.zip)
